### PR TITLE
fix rebased PRs

### LINKED ISSUE #186: Releases being generated even though no-release label is present
**Describe the bug**

In one of our repos we're seeing consistent deployments when we expect no deploy to happen. This churn specifically happens when @renovatebot makes a pull request to update dependencies, so it _may_ be how it's handling the commit/merge process. 

Here's an [example PR](https://github.com/artsy/renovate-config/pull/148) where the no-release label is present (`Version: Trivial` in our case) but the release is still happening. Here's the [CI results](https://circleci.com/gh/artsy/renovate-config/991) of that given merge. 

@hipstersmoothie I'll update with a verbose mode run when I get a chance. If I can chase it down to something particular I'll open a PR. 

**To Reproduce**

I haven't quite identified what the commonality here is. This is the only place in our org where @renovatebot auto-merges code updates so it could be related to that. 

**Expected behavior**

No release should be triggered.

### PR DESCRIPTION
**Pull request by hipstersmoothie**
# What Changed

Do a search for PRs merged since the last release and try to match commits them.

# Why

rebased PRs sometimes (?) don't have a pr in the commit message so none of the labels on the PR would be respected since it was never found.

fixes #186 

Todo:

- [ ] Add tests
- [ ] Add docs
- [ ] Add yourself to contributors (run `yarn contributors:add`)

### CODE REVIEW

File: src/utils/exec-promise.ts [hunk 1]
```diff
@@ -47,8 +47,12 @@ export default async function execPromise(cmd: string, args?: string[]) {
         // The command bailed for whatever reason, print a verbose error message
         // with the stdout underneath
         let appendedStdErr = '';
-        appendedStdErr = allStdout.length ? `\n\n${allStdout}` : '';
-        appendedStdErr = allStderr.length ? `\n\n${allStderr}` : '';
+        appendedStdErr += allStdout.length ? `\n\n${allStdout}` : '';
+        appendedStdErr += allStderr.length ? `\n\n${allStderr}` : '';
+
+        console.log(allStdout, allStderr);
```
Comment (adierkens) [inline src/utils/exec-promise.ts]: use logger?

File: src/utils/load-plugins.ts [hunk 2]
```diff
@@ -29,10 +37,10 @@ export default function loadPlugin([pluginPath, options]: [
   if (isSupported(pluginPath)) {
     plugin = plugins.get(pluginPath);
   } else {
-    try {
-      plugin = require(pluginPath);
-    } catch (error) {
-      plugin = require(path.join(process.cwd(), pluginPath));
+    plugin = tryRequire(pluginPath);
```
Comment (adierkens) [inline src/utils/load-plugins.ts]: should we log a warning if we can't load a plugin?

File: src/utils/exec-promise.ts (hunk 1 above)
Comment (hipstersmoothie) [reply src/utils/exec-promise.ts]: oops consoles shouldn't be there

Comment (hipstersmoothie) [reply src/utils/exec-promise.ts]: removed

File: src/utils/load-plugins.ts (hunk 2 above)
Comment (hipstersmoothie) [reply src/utils/load-plugins.ts]: now it will log when no plugin is found

**Pull request merged**